ops/infra/rpc-proxy/README.md

```diff
@@ -0,0 +1,59 @@
+# RPC-Proxy
+
+## rpc-proxy settings
+
+There are only 2 settings currently surfaced through the container.
+1. What methods to allow
+2. The address of the upstream sequencer
+
+Both are configured through environmental variables.
+```
+- name: SEQUENCER
+  value: sequencer:8545
+- name: ETH_CALLS_ALLOWED
```

Comment:
Couldn't a call also be `rollup_` prefixed? In which case `ETH_CALLS_ALLOWED` may be confusing. Also `eth_call` is a type of call, which may be confusing. Maybe a better name is `RPC_METHODS_ALLOWED`

Reply:
Sure, updated!